### DISCUSSION TITLE
Cloudlfare Proxy

### DISCUSSION THREAD

**Discussion by samchouse** [plausible/analytics#731] [Self-Hosted Support] [answered]
Is there a way to have Plausible proxied behind Cloudflare because it exposes my host IP if it's DNS only?

**Comment by ItalyPaleAle** [comment, marked as answer]
That's what I do... just use put your Plausible server behind a Cloudflare CDN. It will work just fine!

**Reply by samchouse** [reply]
![githubgraph](https://user-images.githubusercontent.com/46873232/109402008-57918100-7920-11eb-8831-d6a5f73bd345.png)
As you can see no visits today, yet I did visit. Both the website and analytics are behind the proxy.

**Reply by ItalyPaleAle** [reply]
I'm not really able to help with your specific issue because I don't know how you architected the solution.

In general, you need to:

1. Point a domain to Cloudflare
2. Create an A record that points to your public IP and make sure the cloud is "orange" (proxied through the CDN)
2. After you've done that, you should be able to see your Plausible installation at the address proxied by the CDN
3. Use that domain name in the script tag from Plausible

It definitely can be done, and I'm one of the many who did it; I don't have enough information to be able to tell you why it's not working in your specific case.

**Reply by samchouse** [reply]
I have done all of that with no success. My website seems to not allow loading of the Plausible script (CORS Error). I do not believe this is Cloudflare proxy anymore but it may be something else Cloudflare related.

**Reply by anadero** [reply]
There is no solution to this? I've installed and tested several analytics, yet plausible only has this issue

**Reply by metmarkosaric** [reply]
the instruction above are working fine. you would need to share full details on your individual setup in order for others to be able to help you troubleshoot what may be the problem
